cpp/ps0003/problem 3 p301F Salwa واجب الحساب 2.cpp

```diff
@@ -4,27 +4,22 @@ using namespace std;
 
 int main()
 {
-cout<<"--------------------------------------------" <<endl;
-cout<<"Problem 0003"<< endl;
-
+    // prompting user for number 2/2
     int num;
-
-
-cout<<"enter the number" << endl;
-cin>> num;
-
-if (num % 2==0)
-{
-    cout <<"your number is even \n1";
-}
-else
-{
-    cout <<"your number is odd \n1";
-
-}
-
-
-
-
+    
+    cout << "enter the number" << endl;
+    cin >> num;
+
+    // checking if the number is odd or even 3/3
+    if (num % 2 == 0)
+    {
+        cout << "your number is even \n";
+    }
+    else
+    {
+        cout << "your number is odd \n";
+    }
+    
     return 0;
 }
+// great work 😎 8/10 ⭐

```

Comment:
- [x] program returns `0` as everything is okay
- [x] program is in the correct path
- [x] no syntax errors
- [ ] formatted code to be clear and readable
- [ ] commented some phrases for blocks of code